### ISSUE TITLE
[BUG] Two portraits (new and old) instead of just one (new)

### ISSUE THREAD

**Issue by AltroTenli** [p4535992/foundryvtt-chat-portrait#45]
**Module Version:** v0.9.7

**Bug description**
Once module is enabled, all new chat messages contain two portraits: one from the enabled module (all settings apply to it) and one default (the one that Foundry used before the module was enabled). So the module does not replace the original chat portrait, but adds a new one instead.

**Steps to reproduce the behavior**
1. Enable module.
2. Make any roll.

**Expected behavior**
Chat message contains only one portrait.

**Screenshots**
![image](https://github.com/p4535992/foundryvtt-chat-portrait/assets/904946/195b31aa-dc67-4c91-aca4-c3e71578f468)

**Browser:** Chrome

**Foundry Version:** Version 11 Stable (315)

**Game System:** D&D 5e

**Comment by Altro** [comment]
I've also tried replicating the bug on a new game without any other modules enabled, and the bug replicates successfully. Hence, the issue is not a module conflict.

**Comment by p4535992** [comment]
Not sure about the other settings, but it should be enough  to enable this settings "Disable chat portrait".
If the problem persist i need to see the settings .... you can use this module to pass to me the setitngs: https://foundryvtt.com/packages/forien-copy-environment/

**Comment by Altro** [comment]
The "Disable chat portrait" disables the module-generated portraits and keeps the default ones. Since I downloaded the moded specifically for the module-generated portraits, I'd need the opposite.

Attaching the JSON extracted via that the module you gave.

Also, thank you so much for getting back so quickly! <3

[foundry-settings-export-2024-04-01-18-06-37-family-matters.json](https://github.com/p4535992/foundryvtt-chat-portrait/files/14824971/foundry-settings-export-2024-04-01-18-06-37-family-matters.json)

**Comment by Altro** [comment]
Yes, indeed. The new module portraits are the reason I downloaded the module. And on all of the images in module description there are only module portraits, without system portraits at all. This is the expected behaviour I'm trying to achieve.
e.g. : https://raw.githubusercontent.com/p4535992/foundryvtt-chat-portrait/master/wiki/images/hooks_feature.png

**Comment by jayseaseven** [comment]
I'm seeing the same thing. In this picture I would just ideally want the square icon and not the circle one from default.
![image](https://github.com/p4535992/foundryvtt-chat-portrait/assets/73178576/5ff93542-7449-44cf-b7f0-12e1541dd8ea)

**Comment by p4535992** [comment]
After looking at Dnd5e and Pf2e , I decided not to support the "removal" of the default avatar , as too many conflicts with other modules would arise.

The module still remains supported, for all those systems that still do not support portraits in default chats.

**Comment by JackrabbitSlim68** [comment]
is the above still true, that i cannot remove the default avatar with chat portrait? Im running dnd 5E 5.3.3 V14 and no other modules and i get a duplicate image in chat.
